info/guide/diktat-coding-convention.md

```diff
@@ -2713,6 +2713,16 @@ fun SomeClass.deleteAllSpaces() {
 }
 ```
 
+#### <a name="r6.2.4"></a> 6.2.4 Don't use property length with operation - 1
```

Comment:
You should add description in a specific chapter; the complete guide is then generated from them

Reply:
Ok, done

Reply:
You can remove new text from `diktat-coding-convention.md` completely and only add it into `guide-chapter-6.md`. After that, from `info` directory you can run `./gradlew updateMarkdownDocumentation` to update guide and other files automatically. We don't do it regularly, so there might be a lot of changes...